alpha/apps/kaltura/modules/extwidget/actions/serveFlavorAction.class.php

```diff
@@ -164,7 +164,28 @@ public static function serveLiveMediaSet($durations, $sequences, $playlistStartT
 
 		return $mediaSet;
 	}
-	
+	// if the entry has clip attributes returns the offsets and updated durations, otherwise returns zeros and original durations
+	protected function retrievePlaylistAttributesData($entry, $durations)
+	{
+		$offsets = array_fill(0, count($durations), 0);
+		if ($entry->getMediaType() == PlaylistType::STATIC_LIST && $entry->getOperationAttributes())
+		{
+			$clipAttributes = array_filter($entry->getOperationAttributes(), function($obj) {
```

Comment:
Dynamic function declaration is not supported in php8 so please fix it already as we are moving to php8 shortly.
So to fix it you need declare the function and pass it to the calls.
So something like this:

$func = function($obj) { return $obj instanceof kClipAttributes; };
$clipAttributes = array_filter($entry->getOperationAttributes(), $func);

Reply:
fixed